packages/sdk-vue/src/createFusionAuth/createFusionAuth.ts

```diff
@@ -1,10 +1,12 @@
-import { ref } from 'vue';
+import { Ref, ref } from 'vue';
 import { SDKCore } from '@fusionauth-sdk/core';
 import { FusionAuth, FusionAuthConfig, UserInfo } from '#/types';
 
 import { NuxtUseCookieAdapter } from './NuxtUseCookieAdapter';
 
-export const createFusionAuth = (config: FusionAuthConfig): FusionAuth => {
+export const createFusionAuth = <T extends UserInfo>(
```

Comment:
We want `<T = UserInfo>` not `extends`. I haven't used `extends` much but I believe this would mean `T` is constrained to have all the properties defined on `UserInfo`, which is not what we want.